sdk/src/test/java/io/customer/sdk/commontest/BaseTest.kt

```diff
@@ -0,0 +1,141 @@
+package io.customer.sdk.commontest
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import io.customer.commontest.util.DeviceStoreStub
+import io.customer.commontest.util.DispatchersProviderStub
+import io.customer.sdk.CustomerIOConfig
+import io.customer.sdk.core.module.CustomerIOModule
+import io.customer.sdk.core.util.CioLogLevel
+import io.customer.sdk.core.util.DispatchersProvider
+import io.customer.sdk.data.model.Region
+import io.customer.sdk.data.store.Client
+import io.customer.sdk.data.store.DeviceStore
+import io.customer.sdk.di.CustomerIOComponent
+import io.customer.sdk.di.CustomerIOStaticComponent
+import io.customer.sdk.util.*
+import okhttp3.ResponseBody.Companion.toResponseBody
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import retrofit2.HttpException
+import retrofit2.Response
+
+/**
+ * Base class for a unit test class to subclass. If you want to create integration tests, use [RobolectricTest].
+ * Meant to provide convenience to test classes with properties and functions tests may use.
+ */
+abstract class BaseTest {
```

Comment:
Because all files moved to `test` and we are no longer using `sharedTest`, we need to copy all files twice. One for `androidTest` and other for `test`. This is only for tracking module. Rest of the modules will leverage `commontest` to write common code.